### PR TITLE
New about page

### PR DESCRIPTION
**Pull request by miguepollo**
This is the new about page, give a look and tell me if something needs to be changed.

### CODE REVIEW

File: web/frontoffice/src/routes/about/+page.svelte [hunk 1]
```diff
@@ -44,35 +44,106 @@
...
-        <h3 class="uppercase my-2 font-bold text-sm">Bitcoiner, remnant and shadowy super coder</h3>
+    <a target="_blank" rel="noreferrer" href="/p/npub1c37zfj85qq9g8a00e5hhatdw47u0j4phqf7lxd2068qp6lu9ykpshddaf4">
+        <div class="card rounded-box max-w-full xl:w-full h-full gap-4 px-4 pt-4 pb-1 mb-3 flex-shrink-0 place-items-center items-center shadow-xl bg-base-300 text-white-content rounded hover:scale-125 duration-300">
+            <div class="avatar mask mask-circle h-40 w-40">
+                <img src="/team/MTC.jpg" alt=""/>
+            </div>
+            <div>
+                <div class="w-full text-center">
+                    <div class="text-xl font-extrabold">MTC</div>
+                    <div class="uppercase my-2 font-bold text-sm">Description</div>
```
Comment (ibz) [inline web/frontoffice/src/routes/about/+page.svelte:121]: Need job title here...

File: web/frontoffice/src/routes/about/+page.svelte [hunk 2]
```diff
@@ -44,35 +44,106 @@
...
+    </a>
+    <a target="_blank" rel="noreferrer" href="/p/npub1cjw49ftnxene9wdxujz3tp7zspp0kf862cjud4nm3j2usag6eg2smwj2rh">
+        <div class="card rounded-box max-w-full xl:w-full h-full gap-4 px-4 pt-4 pb-1 mb-3 flex-shrink-0 place-items-center items-center shadow-xl bg-base-300 text-white-content rounded hover:scale-125 duration-300">
+            <div class="avatar mask mask-circle h-40 w-40">
+                <img src="/team/hodlonaut.jpg" alt=""/>
+            </div>
+            <div>
+                <div class="w-full text-center">
+                    <div class="text-xl font-extrabold">Hodlonaut</div>
+                    <div class="uppercase my-2 font-bold text-sm">Description</div>
```
Comment (ibz) [inline web/frontoffice/src/routes/about/+page.svelte:134]: Need job title...

File: web/frontoffice/src/routes/about/+page.svelte [hunk 3]
```diff
@@ -44,35 +44,106 @@
...
+                </div>
+            </div>
+        </div>
+    </a>
+    <a target="_blank" rel="noreferrer" href="/contact">
+        <div class="card rounded-box max-w-full xl:w-full h-full gap-4 px-4 pt-4 pb-1 mb-3 flex-shrink-0 place-items-center items-center shadow-xl bg-base-300 text-white-content rounded hover:scale-125 duration-300">
+            <div>
+                <div class="w-full text-center">
+                    <div class="text-xl font-extrabold">You?</div>
+                    <div class="uppercase my-2 font-bold text-sm">Bitcoiner , remnant and shadowy super coder</div>
```
Comment (ibz) [inline web/frontoffice/src/routes/about/+page.svelte:144]: No space before ","...

File: web/frontoffice/src/routes/about/+page.svelte [hunk 4]
```diff
@@ -44,35 +44,106 @@
...
     </a>
-
-    <a class="group bg-gray-300/20 p-4 rounded hover:scale-150 duration-300" href={null}>
-        <h2 class="text-3xl font-bold my-4">You?</h2>
-        <p class="visible group-hover:visible lg:invisible lg:group-hover:visible text-xl font-bold my-4"><a class="link" href="/contact">Contact us!</a></p>
-        <h3 class="uppercase my-2 font-bold text-sm">Bitcoiner, remnant and shadowy super coder</h3>
+    <a target="_blank" rel="noreferrer" href="/p/npub1c37zfj85qq9g8a00e5hhatdw47u0j4phqf7lxd2068qp6lu9ykpshddaf4">
+        <div class="card rounded-box max-w-full xl:w-full h-full gap-4 px-4 pt-4 pb-1 mb-3 flex-shrink-0 place-items-center items-center shadow-xl bg-base-300 text-white-content rounded hover:scale-125 duration-300">
+            <div class="avatar mask mask-circle h-40 w-40">
+                <img src="/team/MTC.jpg" alt=""/>
```
Comment (ibz) [inline web/frontoffice/src/routes/about/+page.svelte:116]: All other files are lowercase, maybe we should keep it that way for consistency?

**Review by ibz — CHANGES_REQUESTED** [review]

**Comment by miguepollo** [comment]
Yes of course 
image

**Comment by ktecho** [comment]
@miguepollo Also, could you please center the text in the last box so the "you?" and "bitcoiner..." texts are in the middle of the box?

**Comment by miguepollo** [comment]
Ok

**Comment by ktecho** [comment]
@ibz LGTM

**Pull request merged**